src/main/scala/mesosphere/marathon/api/akkahttp/v2/AppsController.scala

```diff
@@ -46,6 +46,7 @@ import mesosphere.marathon.core.task.Task.{ Id => TaskId }
 import PathMatchers._
 import mesosphere.marathon.raml.{ AnyToRaml, AppUpdate, DeploymentResult }
 import mesosphere.marathon.raml.EnrichedTaskConversion._
+import AppsDirectives._
```

Comment:
Why not `import AppsDirectives.{extractTaskKillingMode, TaskKillingMode}`? What is the benefit of the underscore import?

Reply:
No specific reason.